src/test/resources/filesystems.pan

```diff
@@ -47,5 +47,11 @@ unique template filesystems;
     fs["preserve"] = false;
     append(fs);
 
+    # append md test with useexisting
+    fs=value("/system/filesystems/0");
+    fs["block_device"]=format("md/%s", escape("md/myname"));    
+    fs["useexisting_md"] = true;
```

Comment:
@stdweird This is set only by aii-ks for now.. I don't know another way to pass this from there to ncm-lib-blockdevices.